[2023-03-27T18:08:26.532-0400] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality_pipeline.run_producer manual__2023-03-27T22:08:23.571596+00:00 [queued]>
[2023-03-27T18:08:26.539-0400] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality_pipeline.run_producer manual__2023-03-27T22:08:23.571596+00:00 [queued]>
[2023-03-27T18:08:26.539-0400] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-27T18:08:26.539-0400] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-03-27T18:08:26.539-0400] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-27T18:08:26.546-0400] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): run_producer> on 2023-03-27 22:08:23.571596+00:00
[2023-03-27T18:08:26.551-0400] {standard_task_runner.py:55} INFO - Started process 6733 to run task
[2023-03-27T18:08:26.555-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'air_quality_pipeline', 'run_producer', 'manual__2023-03-27T22:08:23.571596+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/pipeline.py', '--cfg-path', '/var/folders/8z/pwbx6b3n3mnd1g1f397fv1100000gn/T/tmpwg6uhqi7']
[2023-03-27T18:08:26.556-0400] {standard_task_runner.py:83} INFO - Job 16: Subtask run_producer
[2023-03-27T18:08:26.585-0400] {task_command.py:388} INFO - Running <TaskInstance: air_quality_pipeline.run_producer manual__2023-03-27T22:08:23.571596+00:00 [running]> on host srivardhans-air.starry-inc.net
[2023-03-27T18:08:26.616-0400] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yourname
AIRFLOW_CTX_DAG_ID=air_quality_pipeline
AIRFLOW_CTX_TASK_ID=run_producer
AIRFLOW_CTX_EXECUTION_DATE=2023-03-27T22:08:23.571596+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-27T22:08:23.571596+00:00
[2023-03-27T18:08:26.618-0400] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/8z/pwbx6b3n3mnd1g1f397fv1100000gn/T
[2023-03-27T18:08:26.618-0400] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /Users/srivardhan/Desktop/Masters/Masters Project/Kafka Implementation/kafka-pipeline-examples/producer-consumer/producer.py']
[2023-03-27T18:08:26.627-0400] {subprocess.py:86} INFO - Output:
[2023-03-27T18:08:26.650-0400] {subprocess.py:93} INFO - python3: can't open file '/Users/srivardhan/Desktop/Masters/Masters': [Errno 2] No such file or directory
[2023-03-27T18:08:26.652-0400] {subprocess.py:97} INFO - Command exited with return code 2
[2023-03-27T18:08:26.658-0400] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/srivardhan/opt/anaconda3/envs/kafka-pipeline/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-03-27T18:08:26.660-0400] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=air_quality_pipeline, task_id=run_producer, execution_date=20230327T220823, start_date=20230327T220826, end_date=20230327T220826
[2023-03-27T18:08:26.666-0400] {standard_task_runner.py:105} ERROR - Failed to execute job 16 for task run_producer (Bash command failed. The command returned a non-zero exit code 2.; 6733)
[2023-03-27T18:08:26.702-0400] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-27T18:08:26.717-0400] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
